package.json

```diff
@@ -15,6 +15,7 @@
     "postinstall": "cd fhir && npm install",
     "start": "export NODE_ENV=production; cd fhir && node src/index.js;"
```

Comment:
might you want to use `cross-env` here too?

Reply:
Per Jon, the outer package json is for heroku (not for our mongo server) and so won't be running in a Windows environment. Jon will likely restructure these folders to eliminate the need for two package jsons.